src/cmdline.rs

```diff
@@ -274,6 +282,15 @@ pub fn try_parse() -> Result<Command> {
                 Ok(Command::DistAuth)
             } else if matches.get_flag("dist-status") {
                 Ok(Command::DistStatus)
+            } else if matches.get_flag("upload-cache") {
```

Comment:
Hi, I'm considering whether we can add this feature without altering the existing workflow. For instance, we could introduce a new configuration value in sccache, such as `remote_cache`. When the sccache daemon starts, it would load from the remote cache and then write back to it upon shutdown or at regular intervals.

This approach would eliminate the need for users to manually download or upload the cache.

What do you think?

Reply:
In this design, we just need to add a new input in `sccache-action` without changing the workflow.

Reply:
I can definitely see the "upon shutdown" version work and I could possibly already use it in this PR. I don't think it could work for regular intervals since GHA for example does not allow re-uploading to the same cache version+path.

Reply:
> I can definitely see the "upon shutdown" version work and I could possibly already use it in this PR. I don't think it could work for regular intervals since GHA for example does not allow re-uploading to the same cache version+path.

You are right. I'm guessing `regular intervals` is not that useful too. We can remove this part.

Reply:
> I can definitely see the "upon shutdown" version work and I could possibly already use it in this PR.

The key point I want to emphasize is not to add new command lines or alter the existing workflow that utilizes sccache. WDYT?

Reply:
Yep, agreed, I think it's a better design anyway. :)